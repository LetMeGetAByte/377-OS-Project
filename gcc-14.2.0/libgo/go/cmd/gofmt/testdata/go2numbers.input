version https://git-lfs.github.com/spec/v1
oid sha256:a5a162f7f39ffa14dc7caec0741a2375c9fbf7d28beb98d0ba30cd84f92db9cf
size 2004
