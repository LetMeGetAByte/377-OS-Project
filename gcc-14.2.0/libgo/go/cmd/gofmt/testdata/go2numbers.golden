version https://git-lfs.github.com/spec/v1
oid sha256:70592f707a6841a2e9f3106a8d894e37cbedc3b113a771663e6c062221917062
size 1945
